Makefile.in

```diff
@@ -1,17 +1,26 @@
-STRIP=@STRIP@
-OBJCOPY=@OBJCOPY@
-OBJDUMP=@OBJDUMP@
+STRIP ?= @STRIP@
+OBJCOPY ?= @OBJCOPY@
+OBJDUMP ?= @OBJDUMP@
```

Comment:
I like it. I get more frustrated when i set `CC` and it doesn't effect a Makefile than when I forget I have it set and it compiles for the wrong target until I unset it.